src/charm.py

```diff
@@ -509,7 +532,8 @@ def _on_update_status(self, event: UpdateStatusEvent):
         # Cannot check more advanced MongoDB statuses if mongod hasn't started.
         with MongoDBConnection(self.mongodb_config, "localhost", direct=True) as direct_mongo:
             if not direct_mongo.is_ready:
-                self.unit.status = WaitingStatus("Waiting for MongoDB to start")
+                # self.unit.status = WaitingStatus("Waiting for MongoDB to start")
```

Comment:
```suggestion
```

Reply:
Indeed, I forgot to remove this line.
Done!